### ISSUE TITLE
Adding a chapter with system specific tips

### ISSUE THREAD

**Issue by larsenwork** [fontforge/designwithfontforge.com#93]
I know there are some few tips on the download pages but thought we could aggregate them all in a chapter here if deemed relevant.

**Comment by davelab6** [comment]
Maybe even move them here and add a link

**Comment by larsenwork** [comment]
where is "here"? and add link where? :)

**Comment by davelab6** [comment]
Move the info to this repo.

Add a link on the download pages to where they end up on this repo

**Comment by larsenwork** [comment]
to wiki pages?

**Comment by larsenwork** [comment]
like so: https://github.com/fontforge/designwithfontforge.com/wiki

**Comment by davelab6** [comment]
No, I actually dislike wiki pages. Better to use gh-pages in all cases

**Comment by larsenwork** [comment]
so by this repo you mean a chapter in the book as suggested? :)

**Comment by davelab6** [comment]
Yes

**Comment by larsenwork** [comment]
gotcha:)

**Comment by larsenwork** [comment]
boom: https://github.com/fontforge/designwithfontforge.com/pull/97
